src/main/scala/com/activegrid/utilities/Connection.scala

```diff
@@ -0,0 +1,14 @@
+package com.activegrid.utilities
+
+import org.neo4j.driver.v1.{AuthTokens, Driver, GraphDatabase}
+
+/**
+  * Created by nagulmeeras on 29/09/16.
+  */
+object Connection {
+  def getConnection(): Driver ={
```

Comment:
Since the driver will be shared by the whole app, don't create a method for it. Just initialise it as a val.